autoload/mysql.vim

```diff
@@ -11,18 +11,20 @@ endfunction
 
 "This is ran when press 'K' on an SQHTable buffer"
 function! mysql#DescribeTable(table)
-    let l:query = 'DESCRIBE ' . mysql#GetDatabase() . '.' . a:table
+    let l:query = mysqlQuery#GetDescribeTableQuery(mysql#GetDatabase(), a:table)
     call sqhell#InsertResultsToNewBuffer('SQHUnspecified', mysql#GetResultsFromQuery(l:query), 1)
 endfunction
 
 "This is ran when we press 'e' on an SQHTable buffer
 function! mysql#ShowRecordsInTable(table)
-    let l:query = 'SELECT * FROM ' . mysql#GetDatabase() . '.' . a:table . ' LIMIT ' . g:sqh_results_limit
+    let w:table = a:table
```

Comment:
Unfortunately this needs to be `g:table` :(
I added an option for the results of a buffer to split, so now those `w` variables only work on the default option of not splittting